Library/Homebrew/version.rb

```diff
@@ -179,6 +179,19 @@ def self.detect(url, specs)
     end
   end
 
+  # TODO add tests if method is going to be left
+  def self.parse_version(val)
```

Comment:
Just realized this actually is part of the `Version` class and the only difference to my suggestion is the method name. Sorry for the noise and please ignore my stupid comment!